pkgs/by-name/li/libiff/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "libiff";
+  version = "0.1-unstable-2024-03-02";
+  src = fetchFromGitHub {
+    owner = "svanderburg";
+    repo = name;
```

Comment:
Could you use `"libiff"` instead of the variable? That way, it will not be broken if we change the name.